modules/ROOT/pages/7.6.0-release-notes.adoc

```diff
@@ -151,12 +151,14 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[bug-fixes]]
 == Bug fixes
 
-{productname} {release-version} also includes the following bug fix<es>:
+{productname} {release-version} also includes the following bug fixes:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Removed title attribute from dialog tree elements as they already have a tooltip.
+// #TINY-11470
 
-// CCFR here.
+Previously, an issue was identified where custom tooltips for tree structures triggered both the browser's native tooltip (via the `title` attribute) and the custom tooltip simultaneously. This resulted in two tooltips with the same text being displayed, leading to a confusing user experience. 
```

Comment:
´custom tooltips for tree structures triggered both the browser's native tooltip (via the `title` attribute) and the custom tooltip simultaneously´
This seems a bit confusing to me, but I'm not sure how to put it better. There's the custom tooltip ( the text ) that we want to show, the browser's native tooltip functionality ( triggered by title ) which will show the text and the tooltip functionality which tiny itself provides which will show the text.

Reply:
Suggestion @HAFRMO 
```suggestion
Previously, an issue was identified with tooltips for tree structures where multiple tooltip systems displayed the same text simultaneously:

. The browser's native tooltip, triggered by the `title` attribute, displayed the text.
. {productnames}'s custom tooltip functionality also displayed the text.

As a consequence, this caused two tooltips with identical content to appear at the same time, leading to a confusing user experience, where the desired behavior was to show only the custom tooltip provided by {productname}.
```